jsonrpc-specs/src/methods/discv5.json

```diff
@@ -0,0 +1,305 @@
+        "$ref": "#/components/contentDescriptors/Enr"
+      }
+    ],
+    "result": {
+      "$ref": "#/components/contentDescriptors/SendPingResult"
+    }
+  },
+  {
+    "name": "discv5_sendPong",
+    "summary": "Send a PONG message to the specified node",
+    "params": [
+      {
+        "$ref": "#/components/contentDescriptors/Enr"
+      },
+      {
+        "$ref": "#/components/contentDescriptors/RequestId"
+      }
+    ],
+    "result": {
+      "$ref": "#/components/contentDescriptors/SendPongResult"
+    }
+  },
+  {
+    "name": "discv5_sendFindNodes",
+    "summary": "Send a FINDNODES request to a peer, to search within the given set of distances",
+    "params": [
+      {
+        "$ref": "#/components/contentDescriptors/Enr"
+      },
+      {
+        "$ref": "#/components/contentDescriptors/Distances"
+      }
+    ],
+    "result": {
+      "$ref": "#/components/contentDescriptors/SendFindNodesResult"
+    }
+  },
+  {
+    "name": "discv5_sendNodes",
+    "summary": "Respond to a specific FINDNODES request with a NODES response.",
+    "params": [
+      {
+        "$ref": "#/components/contentDescriptors/Enr"
+      },
+      {
+        "$ref": "#/components/contentDescriptors/Nodes"
+      },
+      {
+        "$ref": "#/components/contentDescriptors/RequestId"
+      }
+    ],
+    "result": {
+      "$ref": "#/components/contentDescriptors/SendNodesResult"
+    }
+  },
+  {
+    "name": "discv5_sendTalkRequest",
+    "summary": "Send a TALKREQUEST request with a payload to the given peer.",
+    "params": [
+      {
+        "$ref": "#/components/contentDescriptors/Enr"
+      },
+      {
+        "$ref": "#/components/contentDescriptors/ProtocolId"
+      },
+      {
+        "$ref": "#/components/contentDescriptors/Discv5Payload"
+      }
+    ],
+    "result": {
+      "name": "sendTalkRequestResult",
+      "description": "Returns the request-id assigned to the request message-data",
+      "schema": {
+        "title": "request-id",
+        "$ref": "#/components/schemas/bytes8"
+      }
+    }
+  },
+  {
+    "name": "discv5_sendTalkResponse",
+    "summary": "Respond to a TALKREQUEST request by sending a TALKRESPONSE response.",
+    "params": [
+      {
+        "$ref": "#/components/contentDescriptors/Enr"
+      },
+      {
+        "$ref": "#/components/contentDescriptors/Discv5Payload"
+      },
+      {
+        "$ref": "#/components/contentDescriptors/RequestId"
+      }
+    ],
+    "result": {
+      "name": "sendTalkResponseResult",
+      "description": "Returns \"true\" upon success.",
+      "schema": {
+        "type": "boolean"
+      }
+    }
+  },
+  {
+    "name": "discv5_ping",
+    "summary": "Send a PING message to the designated node and wait for a PONG response.",
+    "params": [
+      {
+        "$ref": "#/components/contentDescriptors/Enr"
+      }
+    ],
+    "result": {
+      "$ref": "#/components/contentDescriptors/PingResult"
+    }
+  },
+  {
+    "name": "discv5_findNodes",
```

Comment:
To be more in line with the discv5 spec terminology, we should probably name it `discv5_findNode`: https://github.com/ethereum/devp2p/blob/master/discv5/discv5-wire.md#findnode-request-0x03